docs/examples/promise/04_generate_account/README.md

```diff
@@ -0,0 +1,5 @@
+# Generate accounts
```

Comment:
This example does not belong here at all, it is not API functionality - we don't even import the API in the example at all. Remove.